proto/ignition/msgs/bounding_boxes.proto

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+syntax = "proto3";
+package ignition.msgs;
+option java_package = "com.ignition.msgs";
+option java_outer_classname = "BoundingBoxesProtos";
+
+/// \ingroup ignition.msgs
+/// \interface BoundingBoxes
+/// \brief Bounding boxes msg
+
+import "ignition/msgs/header.proto";
+import "ignition/msgs/bounding_box.proto";
+
+message BoundingBoxes
```

Comment:
We've been using the `_v` convention for messages that hold an array of another message:

```
double_v.proto
entity_factory_v.proto
float_v.proto
int32_v.proto
int64_v.proto
marker_v.proto
model_v.proto
param_v.proto
plugin_v.proto
pose_v.proto
sensor_v.proto
stringmsg_v.proto
uint32_v.proto
uint64_v.proto
visual_v.proto
```

Let's use that pattern here too for consistency, instead of a plural

Reply:
Ok will do that